brontide/noise.go

```diff
@@ -477,7 +477,7 @@ func (b *Machine) RecvActOne(actOne [ActOneSize]byte) error {
 	// If the handshake version is unknown, then the handshake fails
 	// immediately.
 	if actOne[0] != HandshakeVersion {
-		return fmt.Errorf("Act One: invalid handshake version: %v, "+
+		return fmt.Errorf("act One: invalid handshake version: %v, "+
```

Comment:
This looks strange with the `One` remaining capitalized.  I agree that all errors should start with a lowercase, but that also means the second word should be lowercase as well.  The same goes for the other cases.